src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1980,6 +1980,34 @@ static unsigned int processSubscriptions
         cacheSemGive(__FUNCTION__, "update lastNotificationTime for cached subscription");
       }
     }
+    else
```

Comment:
Maybe I'm not getting the point, but I'd say that this else block is not needed here. Please take into account my comments at https://github.com/telefonicaid/fiware-orion/pull/3890/files#r667810559

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/3890/commits/49b0c6df2c4d993b4e0fec160026fe0ac0017cb7